src/components/Flows/__tests__/SingleFlowRunHistory.test.tsx

```
@@ -12,41 +13,82 @@
+
+describe('Flow Creation', () => {
   // ================================================================================
   it('renders the form', async () => {
-    (global as any).fetch = jest.fn().mockResolvedValueOnce({
-      ok: true,
-      json: jest.fn().mockResolvedValueOnce({
-        logs: {
-          logs: [{ message: 'log-0-0' }, { message: 'log-0-1' }],
-        },
-      }),
+    const { httpGet } = require('@/helpers/http');
```

ESLint

[error] Require statement not part of import statement.

(@typescript-eslint/no-var-requires)